helm/values.yaml

```diff
@@ -1,7 +1,7 @@
 # configure Connaisseur deployment
 deployment:
   replicasCount: 3
-  image: securesystemsengineering/connaisseur:v2.2.0
+  image: securesystemsengineering/connaisseur:v2.2.1
```

Comment:
Why is there a version bump? This only touches tests, doesn't it?

Reply:
fixed